Updated logzio-dir-path (#301)
logzio/logz-docs: _source/logzio_collections/_log-sources/docker.md

      {% include log-shipping/replace-vars.html listener=true %}

    logzio-dir-path	<span class="required-param"></span>

>     : Logs disk path. All the unsent logs are saved to the disk in this location.	

Reviewer: ```suggestion
    : Unsent logs are saved to this location on the disk.	
```
